admin/includes/functions/general.php

```diff
@@ -318,7 +318,10 @@ function zen_get_system_information($privacy = false)
     $mysql_slow_query_log_status = '';
```

Comment:
hi scott,
could we not change this line to `= '0'` as opposed to `= ''` and then eliminate line 321 altogether?  in fact, based on the current coding, line 321 is no longer needed, but i think keeping the code a little consistent, ie vars being either `1` or `0` makes it a bit more legible.